client/src/components/createVote/VoteDates.tsx

```diff
@@ -16,132 +22,96 @@ export default class VoteDates extends Component<IVoteDatesProps> {
     super(props);
   }
 
-  public voteEndDateHandler = (inputMoment) => {
+  public render() {
+    return (
+      <Row>
+        <Col md={6}>
+          <Row>
+            <Col md={12}>
+              <ControlLabel>Deadline</ControlLabel>
+            </Col>
+          </Row>
+          <Row>
+            <Col md={6}>
+              <FormGroup>
+                <HelpBlock>Date</HelpBlock>
+                <Datetime
+                  inputProps={{ id: "voteDateEnd" }}
+                  closeOnSelect={true}
+                  isValidDate={(current) => {
+                    return current.isAfter(moment().subtract(1, "day"), "minute");
+                  }}
+                  timeFormat={false}
+                  onChange={this.voteEndDateHandler}
+                  value={this.props.voteEndDateTime}
+                />
+              </FormGroup>
+            </Col>
+            <Col md={6}>
+              <FormGroup>
+                <HelpBlock>Time</HelpBlock>
+                <Datetime
+                  inputProps={{ id: "voteTimeEnd" }}
+                  dateFormat={false}
+                  closeOnSelect={true}
+                  onChange={this.voteEndTimeHandler}
+                  value={this.props.voteEndDateTime}
+                />
+              </FormGroup>
+            </Col>
+          </Row>
+        </Col>
+
+        <Col md={6}>
+          <Row>
+            <Col md={12}>
+              <ControlLabel>Results expiry date</ControlLabel>
+            </Col>
+          </Row>
+          <Row>
+            <Col md={12}>
+              <FormGroup>
+                <HelpBlock>
+                  <em>How long will the results be available?</em>
```

Comment:
Other HelpBlocks are not italic, either all of them should be or none (imo none).

```suggestion
                  How long will the results be available?
```